### DISCUSSION TITLE
Notification extension

### DISCUSSION THREAD

**Discussion by martinachtman** [j-kaltes/Juggluco#49] [Ideas]
The value in the Notification Bar is very handy. Unfortunately I am missing some information, would it be possible to add them?
- Delta since last reading 1min or 5min depending on Libre version
- mini blood glucose history/graphic
![Screenshot_20230706_160749_One UI Home](https://github.com/j-kaltes/Juggluco/assets/10817800/889c2f47-104b-4d6f-ab4c-3ca74b74ee19)

As a comparison the notification from xDrip, which I currently use for another sensor (Dexcom).

And of course many thanks for all the development work.

**Comment by j-kaltes** [comment]
Never versions of Juggluco (https://www.juggluco.nl/Juggluco/download.html) contain an arrow in the notification for the direction of change. This is not equal to the difference between the last two value, but is returned by Abbott's "algorithm" based on the last 16 one minute measurements and is also used for the arrow in Juggluco itself and in Abbotts Libre apps.
![Notification](https://github.com/j-kaltes/Juggluco/assets/72013079/d4cd7129-f4be-46db-bc3e-6c202dbaa16f)

**Reply by martinachtman** [reply]
Thanks, that's sufficient I've upgraded to version 5.1.3
